pandas/core/indexes/multi.py

```diff
@@ -822,6 +823,12 @@ def labels(self):
         )
         return self.codes
 
+    def _coerce(self, array_like, categories, copy=False):
```

Comment:
See other comment; this was previously `_ensure_frozen` but renamed and removed the `.view(FrozenNDArray)`

FWIW I didn't look too deep into what this actually does so could maybe be simplified further or done away with altogether, but keeping simple for now

Reply:
Let's leave it as is for now.  Can always trim down in a follow-up.